### PR TITLE
Add Config::load_bytes for user to include configs in binary

### PR DESCRIPTION
**Pull request by DoumanAsh**
I'd like to include some of my configs in binary instead of loading them at runtime so proposing this change.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * Added configurable width for debug lines. ([#1016])
 * Added `TextureMetadata::srgb_scale()` for default texture metadata with nearest filter. ([#1023])
 * Added motivation to use Amethyst over gluing the building blocks yourself in the book. ([#1057])
+* Added `Config::load_bytes` for reading configuration from raw bytes.
```
Comment (magnonellie) [inline docs/CHANGELOG.md]: You're missing a link to this PR

Comment (DoumanAsh) [reply docs/CHANGELOG.md]: Added

Comment (magnonellie) [reply docs/CHANGELOG.md]: You also need to add the actual link further down :wink:

Comment (DoumanAsh) [reply docs/CHANGELOG.md]: Oh damn sorry, I forget that it is markdown, not magic. Done

**Comment by torkleyy** [comment]
I think at some point we'll want to have some options loaded dynamically only during development and include them in the binary on deploy.

**Comment by Moxinilian** [comment]
@torkleyy This is something that will probably be handled by the asset pipeline.

**Comment by Moxinilian** [comment]
LGTM!
bors r=Moxinilian,Rhuagh,torkleyy

**Pull request merged**